### DISCUSSION TITLE
Bug - Assertion failure caused by exception during pre-start

### DISCUSSION THREAD

**Discussion by sadekmunawar** [apache/pekko#1383] [General]
Hi Pekko Team,

Are there any plans to fix this bug in Pekko?
https://discuss.lightbend.com/t/expected-behavior-when-child-actor-throws-an-exception-during-pre-start/10753

Thank you

**Comment by pjfanning** [comment]
We don't follow Akka issues. It's a legal minefield due to their licensing. I'll turn this into an issue.

**Reply by sadekmunawar** [reply]
Thanks
